velox/functions/sparksql/tests/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_executable(
   RegexFunctionsTest.cpp
   SizeTest.cpp
   SortArrayTest.cpp
+  SparkBitwiseTest.cpp
```

Comment:
remove Spark prefix from the file name for consistency

Reply:
Removed the prefix. Thanks!